packages/web/app/package.json

```diff
@@ -8,7 +8,10 @@
     "build-storybook": "storybook build",
     "build:config": "tsup-node --no-splitting --out-dir . --loader \".mts=ts\" --format esm --target node21 next.config.mts",
     "dev": "pnpm build:config && next dev | pino-pretty",
+    "generate-changelogs": "pnpm prettier --write ../../../scripts/generate-changelogs.js",
     "postbuild": "pnpm --filter @hive/app deploy --prod --no-optional deploy-tmp/web && rimraf dist/node_modules && mv deploy-tmp/web/node_modules dist && rimraf deploy-tmp/web",
+    "pre:generate-changelogs": "node ../../../scripts/generate-changelogs.js",
```

Comment:
I would group generate-changelogs and pre:generate-changelogs into a single command